modules/client-configuration/pages/client-upgrades-product-migration.adoc

```diff
@@ -8,7 +8,9 @@ You can also migrate openSUSE Leap to a later minor version or to the correspond
 
 * openSUSE Leap 15.1 to 15.2, or
 * openSUSE Leap 15.1 to {sle} 15 SP1, or
-* openSUSE Leap 15.2 to {sle} 15 SP2
+* openSUSE Leap 15.2 to {sle} 15 SP2, or
+* openSUSE Leap 15.3 to {sle} 15 SP3, or
+* openSUSE Leap 15.4 to {sle} 15 SP4
```

Comment:
```suggestion
* openSUSE Leap 15.4 to {sle} 15 SP4
```
Here we only want the minimally needed examples.
Maybe, we'd use opensuse leap 15.3 as the starting point now.